recipes/tinyxml2/all/patches/0001-cmake4-support.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4ec836b..5d6a716 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,12 +5,15 @@ IF(BIICODE)
+     ENDIF()
+     RETURN()
+ ENDIF(BIICODE)
+-cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
++cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+ cmake_policy(VERSION 2.6)
+ if(POLICY CMP0063)
+-	cmake_policy(SET CMP0063 OLD)
++	# cmake_policy(SET CMP0063 OLD)
+ endif()
+ 
++set(CMAKE_CXX_VISIBILITY_PRESET hidden)
```

Comment:
I may be missing something, why are these two lines being "moved" here, when they are already present in the CMakeLists? (they are the last two lines of this very patch) - I can't find any justification for moving them, as they should have the same in terms of the produced binaries? (perhaps it influences the compiler checks done by `project()`, if anything)

Given that this is versions 8.0.0 and 7.0.0 from 5+ years ago, if the newer versions already support CMake 4, I would be inclined to just not patch them at all.